ripple/src/main/webapp/businessdetails.html

```diff
@@ -89,16 +89,88 @@
         <div class="inline-block-child"><div class="vl"></div></div>
         <p class="inline-block-child" id="bd-open-close">Open</p>
         <div class="inline-block-child"><div class="vl"></div></div>
+        <i class="inline-block-child fa fa-phone fa-lg" aria-hidden="true"></i>
         <p class="inline-block-child" id="bd-phone-number">(323) 656-6115</p>
+        <div class="inline-block-child"><div class="vl"></div></div>
+        <a class="inline-block-child hyper-link" id="business-website" href="http://mandarette.com/">mandarette.com</a>
       </div>
       <!-- tags TODO: SMRUTHI CREATE + DYNAMICALLY LOAD TAGS -->
       <div class="bd-tags">
       </div>
       <!-- ./tags -->
       <h1 class="bd-header" id="bd-header">Mandarette Cafe</h1>
+      <!-- Ratings & price-->
+      <div>
+        <!-- Stars-->
+        <span id="star-1" class="fa fa-star star-color fa-lg"></span>
+        <span id="star-2" class="fa fa-star star-color fa-lg"></span>
+        <span id="star-3" class="fa fa-star star-color fa-lg"></span>
+        <span id="star-4" class="fa fa-star-half-o star-color fa-lg"></span>
+        <span id="star-5" class="fa fa-star-o star-color fa-lg"></span>
+        <!-- ./Stars -->
+        <p class="inline-block-child  bd-left-padding"><span id="bd-number-reviews">321</span> reviews</p>
+        <div class="inline-block-child"><div class="vl"></div></div>
+        <p class="inline-block-child" id="bd-price">$$</p>
+      </div>
+      <!-- Button to open the modal -->
+      <button type="button" id="review-button" class="primary-fill-chip chip-box-shadow bd-subheader-padding-top" data-toggle="modal" data-target="#bdModal">
+        Add a review
+      </button>
+      <!-- The Modal -->
+      <div class="modal show fade" id="bdModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <!-- Modal header -->
+            <div class="modal-header center">
+              <h3 class="col-12 modal-title bd-center-align">
+                Write a review
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </h3>
+            </div>
+            <!--/Modal  -->
+            <!-- Modal body -->
+            <div class="modal-body">
+              <div class="bd-review-subwrapper">
+                <!-- User rates stars -->
```

Comment:
Could more padding also be added around the text box and between the star/dollar ratings and text box?

Reply:
Fixed.